ci/pkl-pipelines/pay-dev/e2e-helpers.pkl

```diff
@@ -0,0 +1,707 @@
+amends "package://pkg.pkl-lang.org/github.com/alphagov/pkl-concourse-pipeline/pkl-concourse-pipeline@0.0.1#/Pipeline.pkl"
+import "package://pkg.pkl-lang.org/github.com/alphagov/pkl-concourse-pipeline/pkl-concourse-pipeline@0.0.1#/TaskConfig.pkl"
+
+import "../common/pipeline_self_update.pkl"
+import "../common/shared_resources.pkl"
+
+resources = new {
+  pipeline_self_update.PayPipelineSelfUpdateResource("pay-dev/e2e-helpers.pkl", "master")
+
+  shared_resources.payCiGitHubResource
+
+  (shared_resources.payGithubResourceWithBranch("e2e-helpers-pipeline", "pay-ci", "master")) {
+    source { ["paths"] = new Listing<String> { "ci/pipelines/e2e-helpers.yml" } }
+  }
+
+  (shared_resources.payGithubResourceWithBranch("endtoend-git-release", "pay-endtoend", "master")) {
+    source { ["tag_regex"] = "alpha_release-(.*)" }
+  }
+
+  (shared_resources.payGithubResourceWithBranch("stubs-git-release", "pay-stubs", "master")) {
+    source { ["tag_regex"] = "alpha_release-(.*)" }
+  }
+
+  (shared_resources.payGithubResourceWithBranch(
+    "reverse-proxy-git-release", "pay-scripts", "master")
+  ) {
+    source { ["tag_regex"] = "reverse_proxy_alpha_release-(.*)" }
+  }
+
+  (shared_resources.payECRResource(
+    "endtoend-ecr-registry-test", "govukpay/endtoend", "pay_aws_test_account_id")
+  ) {
+    source { ["tag"] = "latest" }
+  }
+
+  (shared_resources.payECRResourceWithVariant(
+    "endtoend-candidate-ecr-registry-test",
+    "govukpay/endtoend",
+    "pay_aws_test_account_id",
+    "candidate"))
+
+  (shared_resources.payECRResourceWithVariant(
```

Comment:
Is there any reason arguments here are multiline while the others like `payGithubResourceWithBranch()` args are in one line?

Reply:
Style guide says to keep lines under 100 chars.